guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
+The Ember Data library is included by default in Ember apps. 
```

Comment:
Perhaps:

> `ember-data` is included by default for applications generated with `ember-cli`; however, if you do not wish to use `ember-data` it can easily be removed by removing the `ember-data` entry from `package.json`.

Reply:
Seems good! Done.